stubs-shared/src/main/java/org/freedesktop/wayland/generator/api/WaylandCustomProtocol.java

```diff
@@ -51,7 +51,7 @@
 
     /**
      * Specifies the path to the Wayland protocol XML file.
-     * This can be an absolute path, a path relative to the project directory,
+     * This can be an absolute path, a path relative to the {@code wayland.scanner.protocol.root} option,
```

Comment:
Please document this option in the README under the `Wayland Protocols` heading.